File: C#/TCPClient.cs
//Includes
 using System;
 using System.Text;
 using System.Net;
 using System.Net.Sockets;
public class TCPClient{
 public static void Main(){
  byte[] buffer=new byte[256];
  while(true){
   Socket client=new Socket(AddressFamily.InterNetwork,SocketType.Stream,ProtocolType.Tcp);
   IPEndPoint address=new IPEndPoint(IPAddress.Loopback,3721);
   string message=Console.ReadLine()+"\n";
   client.Connect(address);
   client.Send(Encoding.ASCII.GetBytes(message));
   int size=client.Receive(buffer,0,255,0,out SocketError error);
   buffer[size]=0;
   Console.Write(Encoding.ASCII.GetString(buffer));
   client.Close();}}}

File: Javascript/TCPClient.js
const net=require("net")
const stdin=process.openStdin()
stdin.addListener("data",message=>{
 const client=new net.Socket()
 client.connect(3721,"127.0.0.1")
 client.write(message)
 client.on('data',data=>{process.stdout.write(data.toString('ascii'))})
 client.close()})